examples/separate-config/every-feature-single-dest-single-ms.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
```

Comment:
This example shows the single multiselect instance with a single destination. It has a configuration as a separate js file.

It also shows
- optgroups and options without optgroups
- special characters in texts
- moving multiple selected elements up/down